Aptoide/aptoide-client-v8
# Feature/asv 1197 new appc campaings navigation

**What does this PR do?**

This PR aims to change the way we retrieve appc campaing apps. This will use the new V7 wrapper around the new webservice (still in development). Changes include not only the change in ws but also converting the click when the app is clicked in home and converting the download when the app finishes installing;

**Database changed?**

No

**Where should the reviewer start?**

- [ ] CampaignAnalytics.java

**How should this be manually tested?**

To test this right now you will need some rewrites in charles (speak to me and I can export them and give them to you as there are a few of them). After that, check that the apps are being correctly displayed in home and that the conversions are being correctly made.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1197](<https://aptoide.atlassian.net/browse/ASV-1197>)

**Code Review Checklist**
